TinyConstraints/Classes/Constrainable.swift

```diff
@@ -36,12 +36,12 @@ extension TinyView: Constrainable {
     }
 }
 
-extension LayoutGuide: Constrainable {
+@MainActor extension LayoutGuide: Constrainable {
```

Comment:
Not needed as the compiler infers `@MainActor` isolation for all members declared on this extension since `Constrainable` is isolated to the `@MainActor`.

```suggestion
extension LayoutGuide: Constrainable {
```